lib/std/child_process.zig

```diff
@@ -673,6 +760,7 @@ pub const ChildProcess = struct {
             destroyPipe(stderr_pipe);
         };
 
+        var in_child_process = false;
```

Comment:
Why does `comptime var` not work here? stage1 bug?

Reply:
Actually it's meant to be a runtime value.  It prevents a couple cleanup defer statements that are only supposed to run in the parent process and not the child process.